spec/units/scorecard_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'scorecard'
+
+describe Scorecard do
+  subject(:scorecard) { Scorecard.new(frame) }
+  let(:frame) { double('frame', :add_roll => [roll], :complete? => false, number: 1) }
+  let(:roll) { double('roll') }
+
+  it 'is initialised with the first frame' do
+    scorecard = Scorecard.new
+    expect(scorecard.frames[0]).to be_an_instance_of(Frame)
```

Comment:
This line tells me this test is not isolated. Unit tests should not depend on any class that is not the class under test.